pages/docs/tutorials/cli-documantation-validation.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Validate AsyncApi documents with CLI commands'
+description: 'This tutorial will teach you how to validate AsynApi document using CLI commands'
+weight: 115
+---
+## Introduction
+
+When it comes to APIs, validation is inevitable because it helps you keep them well-defined. Validation means testing an API to see if it meets the required functionality, performance, security, and quality standards. It is important to validate an API to ensure that it functions properly. This tutorial will teach you how to validate your AsyncAPI documents using your CLI (Command Line Interface).
+
+## Background context
+
+An AsyncAPI document is a file that defines and describes the various components of a message-based or event-driven API. Event-driven APIs are a type of asynchronous API that allows consumers to subscribe to events of interest. It adheres to an architecture in which events are fired or triggered. These triggers occur when a producer detects a state change caused by a user's device or an action taken by the user. The producer is an application that sense state changes and produces these changes as messages.
+
+AsyncAPI document enables people or machines communicating with one another to comprehend the capabilities of an event-driven API without requiring access to the source code, documentation, or network traffic inspection.
+The document allows developers to define the structure of their API, the format of the API, the events to which consumers can subscribe, and the response they receive when these events are triggered or an action is taken. AsyncAPI documents are represented as JSON objects that follow the JSON.
```

Comment:
```suggestion
The document allows developers to define the structure of their API, the format of their API, the events to which consumers can subscribe, and the response they receive when these events are triggered, or an action is taken. AsyncAPI documents are represented as JSON objects that follow the JSON.
```
- `their` should be better as it has been just used before
- `represented as JSON objects that follow the JSON` that follow JSON `format`? <- Is this correct?

Reply:
@AnimeshKumar923, yes, it is. I was working on this issue, but I got confused with the method to validate AsyncAPI documents using the CLI.

However, I have conducted research that helped me understand the Command Line Interface, and I just want to confirm with @alequetzalli if I can still continue to work on this issue.